src/main/surfrdMod.F90

```diff
@@ -753,20 +826,26 @@ subroutine surfrd_veg_all(begg, endg, ncid, ns)
                ' must also have a separate crop landunit, and vice versa)'//&
                errMsg(sourcefile, __LINE__))
     end if
-    ! Convert from percent to fraction, check sums of nat vegetation add to 1
-    if ( cft_size > 0 )then
-       wt_cft(begg:endg,:) = wt_cft(begg:endg,:) / 100._r8
-       call check_sums_equal_1(wt_cft, begg, 'wt_cft', subname)
-    end if
+    ! Convert from percent to fraction
     wt_lunit(begg:endg,istsoil) = wt_lunit(begg:endg,istsoil) / 100._r8
     wt_lunit(begg:endg,istcrop) = wt_lunit(begg:endg,istcrop) / 100._r8
     wt_nat_patch(begg:endg,:)   = wt_nat_patch(begg:endg,:) / 100._r8
-    call check_sums_equal_1(wt_nat_patch, begg, 'wt_nat_patch', subname)
```

Comment:
Why was this changed from a subroutine call to inline code?

Reply:
It's still a subroutine call, but I moved it a few lines down.

Reply:
Ahhh, good!